seed/shipping-seeder.js

```diff
@@ -0,0 +1,1039 @@
+//requirements
+const mongoose = require('mongoose');
+const Product = require('../models/shipping'); //schema for this collection
+
+//List of items to add to my dev and test databases - product_id 1-100 match product ids from other team member's dbs. The fake data may not match.
+const products = [
```

Comment:
@ttreit  please refactor this to not contain static data but auto-generated from a pipeline.

Reply:
I'm not sure I understand what you mean.

Reply:
i mean to say that your seed file is simply static data with a single insert, and it should be a data pipeline composed of functions which generate this data and then insert into your database